egs2/jtubespeech/tts1/local/data_prep.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Takaaki Saeki
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+dbr=$1
+dbs=$2
+data_dir=$3
+scorethresh=$4
+ROOTDIR="$(cd $(dirname $(dirname $0)); pwd)"
+
+# check arguments
+if [ $# != 4 ]; then
+    echo "Usage: $0 <db_raw> <db_split> <data_dir> <ctcscore_threshold>"
+    echo "e.g.: $0 downloads/jtuberaw downloads/jtubesplit data/all -0.5"
+    exit 1
+fi
+
+set -euo pipefail
```

Comment:
Maybe better to load path here.
```suggestion
set -euo pipefail
. ./path.sh
```